ste/sender-blockBlobFromLocal.go

```diff
@@ -47,7 +47,11 @@ func newBlockBlobUploader(jptm IJobPartTransferMgr, destination string, p pipeli
 
 func (s *blockBlobUploader) Prologue(ps common.PrologueState) (destinationModified bool) {
 	if s.jptm.Info().PreservePOSIXProperties {
+
 		if unixSIP, ok := s.sip.(IUNIXPropertyBearingSourceInfoProvider); ok {
+			// Clone the metadata before we write to it, we shouldn't be writing to the same metadata as every other blob.
```

Comment:
Any unit tests or e2e tests that we could write up for this scenario you think that could have caught this?

Reply:
@adreed-msft we need to do the same for pageBlob too.